src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -120,6 +120,9 @@ class OTARequestor : public OTARequestorInterface, public BDXDownloader::StateDe
         mProviderEndpointId  = endpointId;
     }
 
+    // Called to set optional RequestorCanConsent value provided by Requestor.
+    void SetProviderParameters(bool RequestorCanConsent) override { mRequestorCanConsent.SetValue(RequestorCanConsent); }
```

Comment:
nit: This function sounds more generic than it actually is used for. We should name it more appropriate for what it is doing.